ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -845,6 +845,17 @@ tests:
       ASSISTED_CONFIG: |
         OLM_OPERATORS=cnv
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-lvm
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        OLM_OPERATORS=lvm
+        MASTERS_COUNT=1
+        MASTER_MEMORY=18000
+        MASTER_CPU=9
```

Comment:
the configuration for lvm in test-infra is righty defined here: https://github.com/openshift/assisted-test-infra/blob/master/src/consts/olm_operators.py#L83-L87
but is actually broken by the logic that tries to merge(?) user defined values, trigger defined values and olm defined values: https://github.com/openshift/assisted-test-infra/blob/master/src/assisted_test_infra/test_infra/utils/operators_utils.py#L100-L101

I propose to set the values here in the jobs (if it works), and to follow up with a fix for test-infra later.